### ISSUE TITLE
Allow the possibility to load a zipped last pimcore stable as before.

### ISSUE THREAD

**Issue by Yivan** [pimcore/pimcore#3328]
Hello,

The new installer is really great, but having the possibility to download a zip with all composer dependencies initlialized was great.
So developper could just download the zip and unzipped it instead of calling `COMPOSER_MEMORY_LIMIT=3G composer create-project pimcore/demo-basic .`.

We have developpers working on Windows or MacOS with Docker. Pimcore works great, but now installing using composer is a real pain (take up to 30 min...). And it needs 3Go... ugh.

This issue is about making available again the pimcore-stable.zip you provided as download in the website.
I think it could help people start with Pimcore in some case where composer is not available/slow or developper don't have right in the moment of install 3Go free (think to developpers on laptop for instance).

It will help those people to start with Pimcore, and you won't loose any people wanting to try Pimcore and get stuck on the composer/3 giga requirements.

I really hope my comments make sense, we like the new composer way, but offering the zip as alternative was very helpful in some case.

Thanks.

**Comment by Yivan** [comment]
By the way having a zip alternative we could have bypass temporarily the problem i reported here: https://github.com/pimcore/pimcore/issues/3327 ;-) An old way zip install is always a good fallback.

**Comment by dpfaffenbauer** [comment]
Not sure if that’s the best way to go. Composer already is sort of a industry standard in PHP now, so why not commit 100% to it. And once you installed Pimcore, composer caches all libs anyway, so an Installation will be faster 
In the future (depending on your configuration of course).

Needing 3G of Ram doing that, is already being investigated. And this is only a peak during the install. If you use a custom
Skeleton which also includes a composer.lock, it doesn’t need 3G. So there are a lot of ways to handle that.

**Comment by Yivan** [comment]
@dpfaffenbauer Thanks for your advice. I wasn't aware I can customize skeleton. Sure if composer.lock is already here, composer don't need to check dependencies resolution as this is this one which consume so much memory. I will try this way.
It's great news if this memory poblem is being investigated!

On my laptop i getted stuck because i have only 8 Gig (which is good for a laptop), and increasing Docker to 3Gig i need to reboot my computer to make free mem available, change Docker settings (default value on win/macos is 2048 Mo) and install Pimcore waiting 30 min (the fault to slow CIFS docker share...)... and after reduce memory again to 2048 Mo because i want to keep some free memory for other programs as Pimcore (when installed) don't need those 3Go... here is my experience.

I know it is not a Pimcore issue, purpose of the issue was to keep the zip alternative for friendly install in some use case for newcommer (not just  us), but i understand perfectly @brush closed the issue.
As solution for now, maybe we will do a custom zip available to developpers having laptop or being on docker win/macos so thay can bootstrap a fresh Pimcore for test in just a few min.

Thanks.

**Comment by dpfaffenbauer** [comment]
@Yivan I have a different opinion on this matter. Sure having 8 gigs should be enough. But to be honest, todays dev-stacks take way more than 8 gigs. Today I imported a 12GB database into my docker-image and it tock me 28 gigs of ram todo so. I also ungzipped on the fly, but still. And software like Slack and all the Messangers and Chrome and everything takes a huge amount of memory. So 8 Gigs for a Dev Laptop isn't suitable anymore, but thats my personal experience.
